modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,90 @@
+= Create API Groups and Add API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+When you complete the task of creating an API Group with the help of this topic, you would have created an API Group, the very first version of that group, and the very first group instance. Additionally, you would also have added API instances to the API Group instance.
+
+You can add API instances to the API Group instance based on the environment (Production, Sandbox, or others) in which you create the API Group instance. For example, if you created a API Group instance in Production environment, you can add API instances to the API Group instance only from Production.
+
+You create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+// +
+// Otherwise, you may not find the required organization or business group in the list.
+* API Group Administrator permissions to perform to create API Groups .
+//Group administrators is a permission which is set a business group *BG) level. A group administrator can manage (create, update, delete) all groups of the BG. The permissions algo, gives user VIEW access to all API instances of the BG (so user can add them to the group)
+
+
+[[create-api-groups]]
+== Create API Groups
+
+You create an API Group at the organization level.
+
+To create API Groups:
+
+. From the left menu in API Manager, click *API Groups > Create API Groups*.
++
+The Create New API Group window appears.
```

Comment:
Going forward, omit all these sentences about what appears. We assume the step worked and the following step applies to the result of the one before it.

Reply:
Ok Done! Thanks!